src/raftstore/store/worker/region.rs

```diff
@@ -250,13 +264,16 @@ impl SnapContext {
         timer.observe_duration();
     }
 
-    fn handle_destroy(&mut self, region_id: u64, start_key: Vec<u8>, end_key: Vec<u8>) {
-        info!(
-            "[region {}] deleting data in [{}, {})",
-            region_id,
-            escape(&start_key),
-            escape(&end_key)
-        );
+    fn handle_destroy(&mut self, start_key: Vec<u8>, end_key: Vec<u8>) {
```

Comment:
To distinguish synchronous `handle_apply`, i think you should use another name like `cleanup_range`.

Reply:
I think it's fine, `handle_destroy` is also synchronous.